deep-security-agent/README.md

```diff
@@ -0,0 +1,112 @@
+A collection of playbooks to help deploy and manage the Trend Micro Deep Security Agent.
+
+## Requirements
+
+All of the tasks in this repository require a working Deep Security infrastructure. The key component is the Trend Micro Deep Security Manager. The agents (which these playbooks help you manage) do the heavy lifting but the manager gives the marching orders. 
+
+There are no specific technical requirements beyond a standard Ansible deployment.
+
+
+## Dependencies
+
+No dependency is required.
+
+
+## Usage
+
+A sample playbook:
+
+```yaml
+- hosts: all
+  roles:
+    - role: deep-security.deep-security-agent
```

Comment:
I'm not sure what the exact name after we push to Ansible Galaxy. We may need to change it afterwards.